sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -67,19 +67,40 @@ inline void assign_result(pi_result *ptr, pi_result value) noexcept {
 // Invokes the callback for each event in the wait list. The callback must take
 // a single pi_event argument and return a pi_result.
 template <typename Func>
-pi_result forEachEvent(const pi_event *event_wait_list,
-                       std::size_t num_events_in_wait_list, Func &&f) {
+pi_result forLatestEvents(const pi_event *event_wait_list,
```

Comment:
The comment still saying "for each event"

Reply:
The comment has been adjusted.